src/Learn/Component.svelte

```diff
@@ -9,26 +8,31 @@
     import AlphabetReferenceIcon from '../AlphabetReference/Icon.svelte'
     import Score from '../Score/Component.svelte'
     import TranscriptionInput from '../TranscriptionInput/Component.svelte'
+    import type { Alphabet } from '../Alphabet/alphabet'
+    import type { TranscribedAlphabet } from '../Alphabet/alphabet'
```

Comment:
Let's keep those together plus let's put `import type` to the top and in alphabetic order